modules/client/src/addressList/internal/client/methods.ts

```diff
@@ -436,10 +436,14 @@ export class ClientAddressListMethods extends ClientCore
       return {
         minDuration: parseInt(addresslistVotingPlugin.minDuration),
         supportThreshold: parseFloat(
-          formatEther(addresslistVotingPlugin.supportThreshold),
+          parseFloat(
+            formatEther(addresslistVotingPlugin.supportThreshold),
+          ).toFixed(2),
```

Comment:
Why don't we encapsulate this into a helper? This is verbose and spotting two parseFloat() together is weird

Reply:
Sure, makes sense